server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala

```diff
@@ -165,16 +169,17 @@ class AddressIndexRepository @Inject()(
 
```

Comment:
Does postcode need to be able to use the historical or is it always current only?

Reply:
Yes it does. I missed that as part of the merge conflict resolution. I'll update.